master/reference/kube-controllers/configuration.md

```diff
@@ -20,26 +27,27 @@ etcd access:
 | `ETCD_CERT_FILE`    | The full path to the client certificate file for accessing the etcd cluster. | path
 | `ETCD_KEY_FILE`     | The full path to the client key file for accessing the etcd cluster. | path
 
-The `*_FILE` variables are _paths_ to the corresponding certificates/keys. As such, when the policy controller is running as a Kubernetes pod, you
+The `*_FILE` variables are _paths_ to the corresponding certificates/keys. As such, when the controllers are running as a Kubernetes pod, you
```

Comment:
Similar to the above, I think "a Kubernetes pod" --> "Kubernetes pods"

Reply:
There are multiple controllers within a single pod, so I think this is right.